### PR TITLE
Tests ButtonReset component #446

### PR DESCRIPTION
**Pull request by rob111**

### CODE REVIEW

File: src/test/components/ButtonReset.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { ButtonReset } from '../../forms/ButtonReset';
+
+import { CLIENT_DEFAULTS } from '../../utils/CLIENT_DEFAULTS';
+
+
+describe('Test ButtonReset component', () => {
+  it('Test click event', () => {
+    let mockCallBack = jest.fn();
+
+    let button = shallow(<ButtonReset onClick={mockCallBack} snippets={{}}></ButtonReset> )
```
Comment (knod) [inline src/test/components/ButtonReset.test.js:14]: Just a note for the future - if you wrap this in a `<div>` you can then do `.find('ResetButton')` instead.

**Review by knod — APPROVED** [review]
This looks good to me! Feel free to merge it when you want. I added a note that might be helpful in the future, but you don't need to change anything right now.

File: src/test/components/ButtonReset.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import { ButtonReset } from '../../forms/ButtonReset';
+
+import { CLIENT_DEFAULTS } from '../../utils/CLIENT_DEFAULTS';
```
Comment (turnerhayes) [inline src/test/components/ButtonReset.test.js:7]: It doesn't look like `CLIENT_DEFAULTS` or `renderer` is used in this file. Can these be removed?

**Review by turnerhayes — APPROVED** [review]
This branch has some lint errors, too (although they don't seem to be at all related to this PR--don't know how we generally treat things like that. Separate ticket? @knod )

**Comment by knod** [comment]
@turnerhayes : Yeah, linting is separate.

File: src/test/components/ButtonReset.test.js (hunk 2 above)
Comment (knod) [reply src/test/components/ButtonReset.test.js:7]: Wonder why the tests aren't catching that. It's supposed to catch warnings like 'unused vars' last I checked. Wonder if/when that changed. Regardless, it doesn't break anything.

**Review by turnerhayes — APPROVED** [review]

**Pull request merged**